ember-server/src/test/scala/org/http4s/ember/server/EmberUnixSocketSuite.scala

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2019 http4s.org
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.http4s.ember.server
+
+import cats.syntax.all._
+import cats.effect._
+import org.http4s._
+import org.http4s.ember.client.EmberClientBuilder
+import fs2.io.net.unixsocket.{JnrUnixSockets, UnixSocketAddress}
+import org.http4s.client.middleware.UnixSocket
+import scala.concurrent.duration._
+
+class EmberUnixSocketSuite extends Http4sSuite {
+
+  test("Should be able to connect to a server created") {
```

Comment:
Does this test need to be conditional on platform?  Does this run on Windows?  We've had a handful of contributors on Windows.  I suppose one of them can let us know if it's an issue.

Reply:
Well, JNR doesn't suport windows. https://github.com/jnr/jnr-unixsocket/issues/83

Reply:
And I'll bet JDK17 fails a different way on Windows.

I don't particularly want to get into adding multiple OSes to our build matrix (though we've got another one with the resource service this week...), but I predict within six months we're going to have a contributor ask WTF on this test.  We can guard this test now or guard it then. :smile: